.gitignore

```diff
@@ -3,3 +3,4 @@ examples/.ipynb_checkpoints
 examples/JuliaCon2017/.ipynb_checkpoints/
 docs/build/
 docs/site/
+.vscode/
```

Comment:
This may need  a new line (it's the end of the file); this is noted by github.

Reply:
Fixed, thanks!